src/udev/udev-varlink.c

```diff
@@ -62,7 +91,8 @@ int udev_open_varlink(Manager *m) {
 
         r = varlink_server_bind_method_many(
                         m->varlink_server,
-                        "io.systemd.udev.Ping", vl_method_ping);
+                        "io.systemd.udev.Ping", vl_method_ping,
+                        "io.systemd.udev.SetLogLevel", vl_method_set_log_level);
```

Comment:
actually, this too sounds as if it should be somewhat generic. it's similar to the LogControl1 api we have over dbus after all. 

hence maybe move this too into some generic call io.systemd.service.SetLogLevel()

(the implementation here is specific, as it needs to kill workers and stuff, but the api call should still be defined generically)